src/session/content/chat_history.rs

```diff
@@ -109,6 +110,31 @@ mod imp {
     impl BinImpl for ChatHistory {}
 }
 
+fn subtitle_expression(chat: &Chat) -> gtk::Expression {
```

Comment:
Could you put this function at the end of the file that it does not sit between the `imp` module and the public API?